code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -345,10 +347,12 @@
 		else
 			enemy_tile.consider_pressure_difference(src, difference)
 
-	our_air.react(src)
+	reacting = our_air.react(src)
+	if(our_excited_group)
+		our_excited_group.has_reactions |= reacting
 
 	update_visuals()
-	if(!consider_superconductivity(starting = TRUE) && !active_hotspot) //Might need to include the return of react() here
+	if(!consider_superconductivity(starting = TRUE) && !active_hotspot && !reacting) //Might need to include the return of react() here
```

Comment:
remove my old comment, also if you're gonna check the flags in excited group handling do it here too